CDTDatastore/HTTP/CDTURLSession.h

```diff
@@ -26,7 +26,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface CDTURLSession : NSObject <NSURLSessionDataDelegate>
+@interface CDTURLSession : NSObject <NSURLSessionDataDelegate, NSURLSessionDelegate>
```

Comment:
I think this should be ` NSURLSessionTaskDelegate` instead of `NSURLSessionDelegate`.